pkg/nsx/services/vpc/vpc.go

```diff
@@ -718,6 +718,10 @@ func (s *VPCService) IsLBProviderChanged(existingVPC *model.Vpc, lbProvider LBPr
 		return false
 	}
 	if lbProvider == AVILB {
+		if existingVPC.LoadBalancerVpcEndpoint == nil {
+			log.Error(nil, "LoadBalancerVpcEndpoint is nil, skipping", "VPC path", *existingVPC.Path)
```

Comment:
could we have one info log for all the cases for AVI

Reply:
ok